package.json

```diff
@@ -35,14 +34,14 @@
   },
   "optionalDependencies": {},
   "devDependencies": {
+    "ava": "^1.4.1",
```

Comment:
It seems like we are not using `ava` even though included here, or am I missing something?

Reply:
Will remove it!